### PR TITLE
20160830 fix for deployment

### PR DESCRIPTION
**Pull request by ssneilss**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /* global dbcs:true generateID:true */
 const config = {
 	port: process.argv.includes('--production') ? 80 : 3000,
-	mongoPath: 'mongodb://localhost:27017/Aquaforces',
+	// MongoDB remote ip: 146.148.33.112
+	mongoPath: `mongodb://${process.env.mongoServer || 'localhost:27017'}/Aquaforces`,
```
Comment (ssneilss) [inline server.js:5]: connect the remote MongoDB at 146.148.33.112:27017

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "build:dll": "./node_modules/.bin/webpack --config webpack.dll.js",
     "build": "npm run build:dll && npm run build:webpack",
     "dev": "./node_modules/.bin/webpack --config webpack.config.js --watch --progress",
-    "production": "./node_modules/.bin/webpack -d --optimize-minize",
+    "production": "./node_modules/.bin/webpack -p --optimize-minimize",
```
Comment (ssneilss) [inline package.json:17]: `-d` is for development, should be `-p`

should be`-minimize`

**Comment by alexandernext** [comment]
Are we ready to merge?

**Comment by ssneilss** [comment]
@josh is it ok?

**Comment by jshuaf** [comment]
Merge whenever you're ready.

**Pull request merged**